cli/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "stoat",
+  "version": "0.0.3",
+  "description": "Stoat CLI",
+  "main": "src/index.ts",
+  "bin": {
+    "stoat": "./dist/index.js"
+  },
+  "scripts": {
+    "start": "nodemon --watch 'src/**/*.ts' --exec 'ts-node' src/index.ts",
+    "format": "prettier --write '**/*.{ts,json}'",
+    "format-check": "prettier --check '**/*.{ts,json}'",
+    "lint": "eslint src/**/*.ts",
+    "create": "npm run build && npm run test",
+    "build": "npm run format && ncc build --source-map --license licenses.txt",
+    "local": "npm i -g && stoat",
+    "refresh": "rm -rf ./node_modules ./package-lock.json && npm install",
+    "test": "jest --coverage"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stoat-dev/stoat.git"
```

Comment:
```suggestion
    "url": "git+https://github.com/stoat-dev/stoat-action.git"
```

`stoat` is private.

Reply:
I copied this from the root action package.json, will fix there too